packages/cli/tests/utils/compiler.spec.ts

```diff
@@ -72,10 +61,9 @@ describe('Compiler utils', () => {
                 outputName: outputNameTest,
             })) as unknown as { app: { output: { fileName: string }[] }; settings: { output: { fileName: string }[] } };
 
-            expect(createHash).toHaveBeenCalledWith('sha256');
-            expect(result.app.output[0].fileName).toBe(`${outputNameTest}.${testHash}.js`);
-            expect(result.app.output[1].fileName).toBe(`${outputNameTest}.${testHash}.css`);
-            expect(result.app.output[2].fileName).toBe(`${outputNameTest}.${testHash}.html`);
+            expect(result.app.output[0].fileName).toBe('assets/index-sM5bF_1u.js');
+            expect(result.app.output[1].fileName).toBe('assets/index-B-x5KKng.css');
```

Comment:
🤔 Where does the code in the filename comes from? I'm wondering why it's not `index.js` or `index.css` like the last 2 that have a proper name. (index.html, settings.js)

Reply:
It is after the build, vite puts a unique identifer on the js/css files to break cache.

Reply:
Got it! Thanks.